src/main/java/com/example/fooddelivery/repository/OrderRepository.java

```diff
@@ -1,7 +1,14 @@
 package com.example.fooddelivery.repository;
 
+import com.example.fooddelivery.enums.DeliveryStatus;
 import com.example.fooddelivery.model.Order;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.CrudRepository;
 
-public interface OrderRepository extends JpaRepository<Order, Long> {
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+public interface OrderRepository extends CrudRepository<Order, Long> {
```

Comment:
Why did you changed it to CrudRepository?

Reply:
Because currently, I don't need any feature from `JpaRepository`. I think we need JpaRepository only if we'll use pagination for orders.